.github/workflows/auto-publish.yml

```diff
@@ -29,3 +29,28 @@ jobs:
           verbose: true
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+  release-nwbpanel-image:
+    name: NWB Panel Image
+    runs-on: ubuntu-latest
+    needs: publish-pypi
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Recommend using a static token that matches the specified username to keep them in sync - note that it will need package permissions as well